packages/vsx-registry/src/browser/style/index.css

```diff
@@ -147,6 +147,19 @@
 
 .theia-vsx-extension-editor .body {
     flex: 1;
+    margin: auto;
```

Comment:
@kaiyue0329 I think that we should also include padding as I mentioned in the issue.